src/share/vm/runtime/globals_ext.hpp

```diff
@@ -132,6 +132,10 @@
                                                                             \
   manageable(bool, PrintThreadCoroutineInfo, false,                         \
           "print the park/unpark information for thread coroutine")         \
+                                                                            \
+  manageable(uintx, HumongousObjectAllocationThreshold, 128*M,              \
```

Comment:
Humongous is misleading, change to HugeObjectAllocationThreshold

Reply:
fixed